src/staticmaps.js

```diff
@@ -172,7 +173,7 @@ class StaticMaps {
 
       return z;
     }
-    return null;
+    return this.zoomRange.min;
```

Comment:
`whats the impact of this? Are we sure min is good ? Before it was null ?

Reply:
It crashed if it returned null because the zoom level is not defined in that case. The previous implementation assumed that the levels were all available so that it would always find something (level 1 is the whole world so it should match).

Reply:
The min default is because it didn't find any match on the previous levels so we take the lowest possible (most zoomed out)